lib/src/pdf_page_view.dart

```diff
@@ -70,10 +70,29 @@ class PDFPageView {
     _jsInternal.callMethod('setPdfPage', [pdfPage._jsInternal]);
   }
 
-  void update({num scale = 0, num rotation = 0}) {
+  void update(num scale, [num rotation = 0]) {
+    _updateV2(scale: scale, rotation: rotation);
+
+    if (!_isUpdated(scale: scale, rotation: rotation)) {
```

Comment:
:oooooh: I like this approach! Clever Mr Elliott 🤔

Reply:
Should we do V1 first, and then V2? Since we are currently on 2.5.207? Or since we are hoping to upgrade soon, just allow these to take the few extra milliseconds for each call currently?

Reply:
Yeah I was thinking that if we consumed this interop we'd be planning to change the defaults to at least 2.11 around the same time.